logger/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+#Interface target for logger lib.
+add_library(logger_lib_target INTERFACE )
```

Comment:
Please make sure that the brackets and spaces etc. are made uniform everywhere. It should be -
`add_library(logger_lib_target INTERFACE)`